hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/ListingBasedRollbackHelper.java

```diff
@@ -147,4 +163,15 @@ public ListingBasedRollbackHelper(HoodieTableMetaClient metaClient, HoodieWriteC
     };
     return fs.listStatus(FSUtils.getPartitionPath(config.getBasePath(), partitionPath), filter);
   }
+
+  private FileStatus[] getBaseAndLogFilesFromCommitMetadataToBeDeleted(HoodieCommitMetadata commitMetadata, String partitionPath, FileSystem fs) throws IOException {
```

Comment:
Let's limit this method to just extract files paths from metadata, and then do the listing in its caller.  We can also shorten its name to just `getFilesFromCommitMetadata`